### ISSUE TITLE
GLK Peer Review QA - Issue 22

### ISSUE THREAD

**Issue by gkerber** [Helen1479/FGHJ4567.github.io#25]
22.
Page Title: The Air Can Be Regulated By Weather
Filename/URL:   https://helen1479.github.io/FGHJ4567.github.io/Weather.html
Type of Issue: Validity – Accessibility
Description of Issue:  Page failed accessibility testing:
Accessibility Insights for Web 
2 failures were detected:
html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1
list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (1) WCAG 1.3.1
